slsim/Sources/supernovae.py

```diff
@@ -81,3 +102,30 @@ def get_apparent_magnitude(self, time, band, zpsys="AB"):
         """
 
         return self.bandmag(band, zpsys, time)
+
+    def set_magnitudes(
```

Comment:
can you document this definition?

Reply:
can you also specify in and output variables?

Reply:
Done. I've also changed the name of the function to be a little more clear